java-sdk/example/data/test_cb.json

```diff
@@ -5,7 +5,7 @@
   "comment": "add sharedDBName for sharing a DBs for Users, should be named with lowercase and underscore separator, aka snake case",
   "sharedDBName": "harness_test_shared_users",
   "comment": "modelContainer is output an should be a full directory path on your local fs",
-  "modelContainer": "/Users/pat/big-data/harness/models/",
```

Comment:
@pat if you don't mind

Reply:
if we add this then, since they are only used for int tests, the tests should check they exist or create them

Reply:
@andreyshikov

Reply:
Yes understood but those don't exist on any machine and since these are use only for tests, then the tests can create them if they don't exist

Reply:
I'm not sure it will be created, I created them myself on my laptop

Reply:
@andreyshikov notice that `.../harness/models` and `.../harness/mirrors` are used in every test engine.json so after any reboot or on each new install they will have to be created, why not do this in the tests so the test will just work out of the box, saving us countless hand tasks over time.

Reply:
@pferrel do you mean I should add `mkdir -p /tmp/harness/models /tmp/harness/mirrors` to the cb test.sh?